subiquity/server/server.py

```diff
@@ -574,10 +574,11 @@ async def wait_for_cloudinit(self):
 
     def select_autoinstall(self):
         # precedence
-        # 1. autoinstall at root of drive
-        # 2. command line argument autoinstall
-        # 3. autoinstall supplied by cloud config
-        # 4. autoinstall baked into the iso, found at /cdrom/autoinstall.yaml
+        # 1. command line argument autoinstall
```

Comment:
Please update doc/intro-to-autoinstall.rst with a new section talking about this order